block/utils/init_provider.go

```diff
@@ -57,17 +54,12 @@ func InitProviders(conf *config.Config, logger *zap.Logger) (registry.Providers,
 		return providerRegistry, nil
 	}
 
-	return nil, errors.New("No providers registered")
-}
-
-// isEmptyStringValue ...
-func isEmptyStringValue(value *string) bool {
-	return value == nil || *value == ""
+	return nil, errors.New("no providers registered")
 }
 
 // OpenProviderSession ...
 func OpenProviderSession(conf *config.Config, providers registry.Providers, providerID string, ctxLogger *zap.Logger) (session provider.Session, fatal bool, err error) {
-	return OpenProviderSessionWithContext(nil, conf, providers, providerID, ctxLogger)
+	return OpenProviderSessionWithContext(nil, conf, providers, providerID, ctxLogger) //nolint:staticcheck
```

Comment:
how to fix this?

Reply:
We can pass context.TODO(). It is suggested not to pass nil context.

Reply:
Done